packages/react-instantsearch-dom/src/components/Hits.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createClassNames } from '../core/utils';
+
+type Hit = {
+  objectID: string | number;
+};
+
+type HitProps = {
+  hit: Hit;
+};
+
+type Props = {
+  hits: Hit[];
+  className?: string;
+  hitComponent?: React.FunctionComponent<HitProps>;
+};
+
+const cx = createClassNames('Hits');
+
+const Hits: React.SFC<Props> = ({
+  hits,
+  className = '',
+  hitComponent: HitComponent = DefaultHitComponent,
+}) => (
+  <div className={classNames(cx(''), className)}>
+    <ul className={cx('list')}>
```

Comment:
I thought this was an `ol` now? Did we not do that breaking change on React InstantSearch yet?

Reply:
Nope the change came after the release of the V5 of React InstantSearch.